Author: GeorgeClark <GeorgeClark@0b4bb1d4-4e5a-0410-9cc4-b2b747904278>
Date:   Wed Feb 13 00:49:20 2012 +0000

    Item12391: More validations   This patch applies to Foswiki 1.1.3-1.1.6 post Item12285 patch

~~~PATCH fee73707cdd378ca45e87a47c7198de4:92a80e84b34d0fa442e077ffc2f8a86a lib/Foswiki/Macros/MAKETEXT.pm (Foswiki-1.1.4)
--- lib/Foswiki/Macros/MAKETEXT.pm.post12285	2013-02-13 10:26:42.520780283 -0500
+++ lib/Foswiki/Macros/MAKETEXT.pm	2013-02-13 10:26:51.362682708 -0500
@@ -25,8 +25,8 @@
     $str =~ s/\]/~]/g;
 
     # restore already escaped stuff:
-    $str =~ s/~~\[/~[/g;
-    $str =~ s/~~\]/~]/g;
+    $str =~ s/~~+\[/~[/g;
+    $str =~ s/~~+\]/~]/g;
 
     $max         = 0;
     $min         = 1;

